modules/longhorn/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "helm_release" "longhorn" {
+  name             = "longhorn"
```

Comment:
Assuming the user were using helm directly, rather than going through our add-ons: do they have to use these values for `name` and `namespace` or are they able to specify custom values for those?